packages/web3-core/src/web3_context.ts

```diff
@@ -104,6 +104,7 @@ export class Web3Context<
 			| SupportedProviders<API>
 			| Web3ContextInitOptions<API, RegisteredSubs>,
 	) {
+		console.log(providerOrContext)
```

Comment:
I think its for local debugging so should be removed?

Reply:
Yes this PR is still a draft and not ready for review